### DISCUSSION TITLE
Does vite-plugin-ssr support SOLIDjs per page SSG and SSR?

### DISCUSSION THREAD

**Discussion by MilayaAvalosAguilar** [vikejs/vike#650] [Discussion] [answered]
I'm comming from Solid Start which as of now it is all or nothing with Client Side Rendering or Server Side Rendering, I'm looking forward to work with vite-plugin-ssr because it claims to be platform agnostic and I'd like to use all its features on Solidjs, however VPS seems to focus mainly on Vue and React so I don't know if all features are supported for other frameworks as well. Examples don't help either as they only show SSR and SPA with solid

Thanks in advance

**Comment by brillout** [comment, marked as answer]
See https://github.com/brillout/vite-plugin-ssr/discussions/648
